versioned_docs/version-latest/how-to-guides/increasing-open-file-limit.md

```diff
@@ -20,7 +20,7 @@ First, use lima override.yaml to write the provisioning scripts.
   <TabItem value="macOS">
 
 ```
-~/Library/Application Support/rancher-desktop/lima/_config/override.yaml
+"~/Library/Application Support/rancher-desktop/lima/_config/override.yaml"
```

Comment:
For shell compatibility the `~` must be outside the quotes, otherwise it won't get expanded:

```suggestion
~"/Library/Application Support/rancher-desktop/lima/_config/override.yaml"
```

Alternative:

```suggestion
"$HOME/Library/Application Support/rancher-desktop/lima/_config/override.yaml"
```

Note that there are several other reference that should be changed too then:

```console
$ grep -r "~/Library/Application Support" *
how-to-guides/increasing-open-file-limit.md:~/Library/Application Support/rancher-desktop/lima/_config/override.yaml
how-to-guides/provisioning-scripts.md:~/Library/Application Support/rancher-desktop/lima/_config/override.yaml
references/rdctl-command-reference.md:- macOS: ~/Library/Application Support/rancher-desktop/
tutorials/working-with-webassembly.md:- macOS: `~/Library/Application Support/rancher-desktop/containerd-shims`
ui/snapshots.md:- macOS: `~/Library/Application Support/rancher-desktop/snapshots`
```

And the `$HOME` variant is also used in one place:

```console
$ grep -r "HOME/Library/Application Support" *
how-to-guides/mirror-private-registry.md:`$HOME/Library/Application Support/rancher-desktop/lima/_config/override.yaml`
```

They should probably all use the same format.

The docs are also duplicated in several places, so to keep the change propagating forward it needs to be updated in the `docs/` tree as well. And for consistency `versioned_docs/version-1.16/` should match `versioned_docs/version-latest`.

Let me know if you want me to update your PR instead, as this is just busy-work.

Reply:
D'oh, I totally should have caught the tilde outside of the double quotes.

If you have the bandwidth and/or are working on docs anyway, please feel free to close this PR and address as you see fit. This was more a "whoops, I just created a new file at `~/Library/Application` and wanted to help prevent anyone else from making the same mistake 😅

Reply:
I've created #400 to replace this PR. I decided to add a backslash before the space instead of quoting just part of the string.

I'm going to close this PR; thank you for bringing the issue to our attention!